<?xml version="1.0"?>
<launch>

 <!-- small description about your node -->
 
 <!--<node pkg="my_package_name" type="my_node_name" name="my_node_name"
 respawn="false" output="screen" args="$(find trajectory_planning)/config/my_arg_file.yaml"/>-->
 
 <node pkg="trajectory_planning" type="trajectory_planning_node" name="TrajectoryPlanningNode" respawn="false" output="screen">
	<rosparam command="load" file="$(find trajectory_planning)/config/config_trajectory_planning.yaml"/>
</node>

</launch>
